http/grpc/src/test/java/io/quarkus/ts/http/grpc/GrpcTlsSeparateServerIT.java

```diff
@@ -0,0 +1,63 @@
+package io.quarkus.ts.http.grpc;
+
+import static io.quarkus.test.services.Certificate.Format.PEM;
+
+import org.junit.jupiter.api.AfterAll;
+
+import io.quarkus.test.bootstrap.CloseableManagedChannel;
+import io.quarkus.test.bootstrap.GrpcService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.security.certificate.Certificate.PemCertificate;
+import io.quarkus.test.security.certificate.CertificateBuilder;
+import io.quarkus.test.services.Certificate;
+import io.quarkus.test.services.QuarkusApplication;
+import io.vertx.mutiny.ext.web.client.WebClient;
+
+@QuarkusScenario
+public class GrpcTlsSeparateServerIT implements GRPCIT, StreamingHttpIT, ReflectionHttpIT {
+
+    private static final String CERT_PREFIX = "grpc-tls-separate-server";
+    private static WebClient webClient = null;
+
+    @QuarkusApplication(grpc = true, ssl = true, certificates = @Certificate(prefix = CERT_PREFIX, format = PEM, configureKeystore = true, configureTruststore = true))
+    static final GrpcService app = (GrpcService) new GrpcService()
+            .withProperty("quarkus.profile", "ssl")
+            .withProperty("grpc.client.ca-cert", CertificateBuilder.INSTANCE_KEY, GrpcTlsSeparateServerIT::getClientCaCert)
```

Comment:
I suggest to move CertificateBuilder.INSTANCE_KEY to the caller method, it will make reading the code easier.

That applies to other classes as well

Reply:
Done.